copter/source/docs/autotrim.rst

```diff
@@ -25,54 +25,49 @@ Save trim is the simpler method and involves essentially transferring your radio
 .. image:: ../images/MP_SaveTrim_Ch7.png
     :target: ../_images/MP_SaveTrim_Ch7.png
 
-3. With your CH7 switch in the off position, fly your copter in Stabilize mode and use your transmitters's roll and pitch trim to get it flying level
+3. Find a wind free environment with sufficient space to fly your copter without crashing into something
 
-4. Land and put your throttle to zero
+4. Put the vehicle in Stabilize mode
 
-5. Release the roll and pitch sticks and switch the CH7 switch high for at least 1 second.  The words "Trim saved" should appear in the MP's Flight Data screen's Messages tab
+5. Arm the vehicle and lift into a hover. Flip the switch that you setup to  its high position (>1800us).
```

Comment:
nitpick: there's an extra space between "to" and "its"

Reply:
fixed